aws-rotate-keys/orb.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 description: An orb to rotate AWS keys
 
 orbs:
-  aws-cli: circleci/aws-cli@1
+  aws-cli: circleci/aws-cli@1.0.0
```

Comment:
Technically, it should be safe to specify just the major version, as any 1.x shouldn't have any breaking changes if aws are following semver.

Reply:
it's true - was worried about the default executor changing down the line and that potentially causing issues for people

I'm also thinking about publishing proper semver versions for orbs I contribute to in this repo? In that case you would need to have downstream orbs pinned exactly, to make meaningful updates to your own version

this way if we need to update we could

Reply:
yeah, that sounds sensible - it's better to be safe than sorry 😄